docs/content/Datepicker.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Datepicker
+description: Use datepicker to select a single date, multiple dates, or a range of dates
+status: Alpha
+source: https://github.com/primer/react/blob/main/src/Datepicker/index.tsx
+storybook: '/react/storybook?path=/story/composite-components-datepicker--default-date-picker'
+---
+
+## Default example
```

Comment:
Minor nitpick: let's structure the headings to match what has been defined in https://github.com/primer/react/issues/1701